errors/src/errors/type_checker/type_checker_error.rs

```diff
@@ -983,4 +985,13 @@ create_messages!(
         msg: "The unit type () may appear only as the return type of a function.".to_string(),
         help: None,
     }
+
+    @formatted
+    future_error_member {
+        args: (num: impl Display),
+        msg: format!("Cannot access argument `{num}` from future."),
+        help: Some(
+            "This is likely because the async function is called twice with different future types.".to_string()
```

Comment:
```suggestion
            "Ensure that the async function is not called with multiple times with incompatible types.".to_string()
```
Is this message clearer?